awx/urls.py

```diff
@@ -1,25 +1,27 @@
 # Copyright (c) 2015 Ansible, Inc.
 # All Rights Reserved.
 
-from django.conf.urls import url, patterns, include
+from django.conf.urls import url, include
+from awx.api.urls import scoped_url
+
+urlpatterns = [
+    url(r'', include('awx.ui.urls', namespace='ui', app_name='ui')),
```

Comment:
The `namespace` and `app_name` format for `include()` is deprecated: https://docs.djangoproject.com/en/1.11/topics/http/urls/#namespaces-and-include

Should probably move `app_name` into each `urls.py`.